src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -273,7 +295,7 @@ export const SiteLaunchChecklistBody = ({
                     Site launch tasks ({tasksDone}/{numberOfTasks})
                   </Text>
                 </Td>
-                <Td bg="gray.100" textAlign="center">
+                <Td bg="gray.100" textAlign="center" width="30%">
```

Comment:
take note that we need something to fill the missing 70%

Reply:
sure, for clarity 8a0385d39e653c69de3c4b0ee8cfef3def76ef71